### ISSUE TITLE
Deprecation warning for Babel polyfill@7.4.4

### ISSUE THREAD

**Issue by asheemmamoowala** [mapbox/dr-ui#157]
Using using `yarn install` with dr-ui@0.16.2 in the mapbox-gl-js project, I get the following error multiple times: 

```
@mapbox/dr-ui > @elastic/react-search-ui-views > @babel/polyfill@7.4.4: 🚨 As of Babel 7.4.0, this
package has been deprecated in favor of directly
including core-js/stable (to polyfill ECMAScript
features) and regenerator-runtime/runtime
(needed to use transpiled generator functions):

  > import "core-js/stable";
  > import "regenerator-runtime/runtime";
```

cc @katydecorah

**Comment by katydecorah** [comment]
Thanks for flagging, this should be fixed in #156 . I'll try to follow-up with a release/PR today.

**Comment by katydecorah** [comment]
fixed in https://github.com/mapbox/mapbox-gl-js/pull/8507
